packaging/distros/centos/7/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM flb-build-base-centos/7
```

Comment:
Not very well versed in the project structure so just asking:
There is a folder with a CentOS [Dockerfile](https://github.com/fluent/fluent-bit/blob/master/dockerfiles/Dockerfile.centos7. Will this one replace it?

Reply:
This PR is just pulling a direct copy of the packaging repo rather than any additional changes on top as it's pretty large already and the primary goal is to automate the current process.
These containers are used to build the RPMs/Debs rather than the container images pulled for releases.
That container you linked was actually one I knocked up for a compile check only, you're right they should merge eventually but at the time they were separate.